spec/controllers/concerns/verify_sp_attributes_concern_spec.rb

```diff
@@ -137,7 +137,7 @@
     end
   end
 
-  describe '#needs_completions_screen?' do
+  describe '#needs_completion_screen_reason' do
```

Comment:
This doesn't cover all branches but I wasn't sure how to test the expired or revoked consent cases since I'm less familiar with that code - happy to have someone else finish it up!